strapi/strapi
# enhancement: generate types for media attribute

### What does it do?

- updates `ts:generate-types` command to take into consideration allowedTypes and multiple in the generated types
- updates the Media attribute type to provide a real value instead of any

### How to test it?

- Create or use an app with media attributes, both multiple and not
- Generate types for you Strapi app
- Check if the types match the structure you have build
